src/endian.rs

```diff
@@ -0,0 +1,62 @@
+//! Endian types used in this library.
```

Comment:
What advantage do these have over the `zerocopy` types (https://docs.rs/zerocopy/latest/zerocopy/byteorder/index.html)?

Reply:
I changed to byteorder for explicit endian types. Could you help review this change?